easybuild/easyconfigs/o/OpenImageDenoise/oidn-0.9.0-CrayGNU-19.09.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = "Tarball"
+
+name = 'oidn'
```

Comment:
The directory of the package has to be renamed to `oidn` as https://github.com/eth-cscs/production/pull/1318/files to be consistent with the module name. Otherwise it is not found by EasyBuild.